src/client/components/AchievementComponent/AchieveComponent.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './AchieveComponent.styles.css';
+import keyImage from '../../assets/images/key.png';
+import xImage from '../../assets/images/close-outline1.png';
+import timerImage from '../../assets/images/carbon-timer.png';
+import cupImage from '../../assets/images/cup-icon.png';
+
+export default function AchieveComponent({ achieveTask, Ranking, timeSpent }) {
+  const minutes = Math.floor(timeSpent / 60);
+
+  return (
+    <div className="mainDiv">
```

Comment:
Can you find a better name than mainDiv?

Reply:
I have changed to achieveMain